### ISSUE TITLE
Microphone not working in Flathub version

### ISSUE THREAD

**Issue by matiprogram** [hmlendea/dl-desktop#57]
The Microphone is not working, at least in my fedora Workstation installation.
The gnome shell menu it say me for a second that mic is being used but it close the mic dialog.

**Comment by hmlendea** [comment]
Yeah. Unfortunately it doesn't work in the non-flatpak version either.
It works fine if I load onlinemictest.com instead of duolingo though.

Getting the listening and speaking exercises already requires a workaround, as Duolingo blocks them by default on electron apps.
Until I can find another solution (if it's possible), I am going to remove the current workaround, to avoid confusion.

**Comment by matiprogram** [comment]
This is just an idea, why you don't try make the duolingo app in WebKit (Gnome Web) instead of Electron?
idk if it's possible
